### PR TITLE
Rust support

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Working prototype based on https://github.com/openshift-cloud-functions/faas-rust-runtime

The approach taken is similar to js: copy paste the "runtime code"

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -22,6 +22,8 @@ const (
 	REGISTRY_TLS_VERIFY = "registry_tls_verify"
 	KUBECONFIG          = "kubeconfig"
 	DEBUG               = "kfn_debug"
+	CONFIG				= "config"
+	NAMESPACE			= "namespace"
```
Comment (lance) [inline pkg/config/config.go]: Very minor, but the formatting here is weird.

Comment (slinkydeveloper) [reply pkg/config/config.go]: Solved in next ones :)

File: bootstrap_templates/rust/function.rs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+use serde_json::json;
+
+// Function must have this signature
+pub fn function(event: Option<serde_json::Value>) -> impl futures::Future<Item=Option<serde_json::Value>, Error=actix_web::Error> {
```
[the remark this answers is not in the document]
Comment (slinkydeveloper) [reply bootstrap_templates/rust/function.rs]: well, for now we are doing mainly json inputs... I think we should enforce a single payload encoding protocol in all our images

[the remark this answers is not in the document]
Comment (slinkydeveloper) [reply bootstrap_templates/rust/function.rs]: We should consider discussing about it when we write a function runtime spec

**Pull request merged**